.github/workflows/ci.yml

```diff
@@ -160,19 +160,20 @@ jobs:
       - name: Build
         run: rush build -t sample-static-html-composites
       - name: Visual Regression Tests
+        id: visualregressiontests
```

Comment:
Do we wanna add - between words

Reply:
I just wasn't confident that github actions wouldn't break when doing `steps.visual-regression-tests.outcome` 😅

Reply:
(completing without making this change as keeping this to a v. quick fix and waiting for the ci to run a couple times to test is just too much 😭)